styles/typography/typography-preset-1.json

```diff
@@ -23,7 +23,7 @@
 			},
 			"core/post-title": {
 				"typography": {
-					"fontSize": "var:preset|font-size|xx-large",
+					"fontSize": "var:preset|font-size|xxx-large",
```

Comment:
It looks like this one can be a result of the search and replace. I belive it should stay at `xx-large`.